openedx/core/djangoapps/schedules/templates/schedules/edx_ace/common/base_body.html

```diff
@@ -88,40 +88,40 @@
                                             <a href="{{ social_media_urls.linkedin }}">
-                                                <img src="https://media.sailthru.com/595/1k1/8/o/599f354ec70cb.png"
+                                                <img src="{{ social_media_icons.linkedin }}"
                                                      width="32" height="32" alt="{% blocktrans %}{{ platform_name }} on LinkedIn{% endblocktrans %}"/>
                                             </a>
                                         </td>
                                     {% endif %}
                                     {% if social_media_urls.twitter %}
                                         <td height="32" width="42">
                                             <a href="{{ social_media_urls.twitter }}">
-                                                <img src="https://media.sailthru.com/595/1k1/8/o/599f354d9c26e.png"
+                                                <img src="{{ social_media_icons.twitter }}"
                                                      width="32" height="32" alt="{% blocktrans %}{{ platform_name }} on Twitter{% endblocktrans %}"/>
                                             </a>
                                         </td>
                                     {% endif %}
                                     {% if social_media_urls.facebook %}
                                         <td height="32" width="42">
                                             <a href="{{ social_media_urls.facebook }}">
-                                                <img src="https://media.sailthru.com/595/1k1/8/o/599f355052c8e.png"
+                                                <img src="{{ social_media_icons.facebook }}"
                                                      width="32" height="32" alt="{% blocktrans %}{{ platform_name }} on Facebook{% endblocktrans %}"/>
                                             </a>
                                         </td>
                                     {% endif %}
                                     {% if social_media_urls.google_plus %}
                                         <td height="32" width="42">
                                             <a href="{{ social_media_urls.google_plus }}">
-                                                <img src="https://media.sailthru.com/595/1k1/8/o/599f354fc554a.png"
+                                                <img src="{{ social_media_icons.google_plus }}"
                                                      width="32" height="32" alt="{% blocktrans %}{{ platform_name }} on Google Plus{% endblocktrans %}"/>
                                             </a>
                                         </td>
                                     {% endif %}
-                                    {% if social_media_urls.reddit %}
+                                    {% if social_media_urls.youtube %}
```

Comment:
Why changing from reddit to youtube?

Reply:
We don't have a _gray icon for reddit but we do for youtube :man_shrugging: